### ISSUE TITLE
RagePixel future

### ISSUE THREAD

**Issue by JuhaKiili** [JuhaKiili/RagePixel#2]
I'm new to Github so I'm not sure if anyone sees this or gets notified.

Couple of weeks has gone by without any progress. The reason was that I was in the middle of getting recruited to work at Unity Technologies full-time. I really didn't have motivation to put this project forward when the negotiation was on.

Now that I've decided to work for them, I don't have the luxury of continuing this project full steam. I'd like to see it going forward, but others will have to step up. Otherwise it will never get anywhere.

Here is my blogpost on the matter: http://juhakiili.com/blog/working-for-unity/

**Comment by JoeOsborn** [comment]
Hey, congratulations! I'd be glad to keep hacking on the basics of RagePixel—I'll keep up on this repo and fix any bugs I stumble on—but I can't take on any kind of formal maintainer role.

But I'm really glad that RagePixel exists, even in its limited form, and I'm using it extensively. It's really good!

**Comment by ericraue** [comment]
Congrats on the job! I'm interested in contributing as well. I posted about it (pending moderation) on the new RagePixel forums. Joe, I would be willing to help you maintain the repository. With two of us it won't be as much work. Ideally it would be great to have more people contribute to the project.

**Comment by JuhaKiili** [comment]
Sorry about the forums requiring post approval. I changed this and you can now post normally.

**Comment by JoeOsborn** [comment]
Yes! Eric, I'd be happy to add you to this repo. I have a spritesheet import feature that is actually in this GitHub repo, some stuff for describing animations as sequences of frames rather than ranges, and priority-based animation scheduling. I also wrote an AssetImporter that generates RagePixel sprite data and animations from a directory of spritesheets (rather than having a button in the editor plugin; this is appropriate for my project, but maybe isn't a good general-case solution), but that's not in the repo.

**Comment by JoeOsborn** [comment]
Oh: And if you'd rather keep your changes distinct from the mainline, you can always just fork this repository and send pull requests upstream. Personally though, I think it would be OK for two maintainers to share the same repository.

As for licensing, I think MIT sounds great. @JuhaKiili, what do you think?

I also think it could be handy if issues were reported via GitHub Issues rather than as forum threads, so that we could track them a little easier.

**Comment by JuhaKiili** [comment]
@JoeOsborn MIT sounds fine to me. Just to make sure: MIT allows free use in commercial & non-commercial and with or without included source code?

Where does one typically add the license text in GitHub. Is there a place for it or do I put it into the readme.txt ?

@ericraue I added you as GitHub collaborator

**Comment by JoeOsborn** [comment]
Yeah—MIT license basically says "Please include this license text with the program documentation", 3-clause BSD adds on a term that says "Don't use our organization's name in endorsing or advertising this program". They're both permissive wrt commercial works. It's common to have a `LICENSE` or `COPYING` file or even just cram it into the `readme.txt`.

**Comment by beep2bleep** [comment]
I'm glad to hear this is going open source, I might contribute as this starts to take off.  I was looking at building a pixel perfect collision system myself.

**Comment by ericraue** [comment]
@JuhaKiili Thanks for adding me. Until we have a roadmap I will fork the project and send pull requests so they can be discussed before merged. I'm still pretty new to contributing to open source projects.

@JoeOsborn I've been using this repo since it started so I didn't realize how many features/fixes you already added! Let me know how I can help out.

Either BSD or MIT sound fine for the license.

**Comment by JuhaKiili** [comment]
@ericraue Don't worry about being new. This is my first ever collaboration on anything code related and I don't even know how to use git... :)

**Comment by TheNewb** [comment]
will there be or is there UnityScript(ActionScript) support?
